src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/MaintenanceController.php

```diff
@@ -42,8 +42,10 @@ class MaintenanceController extends FrameworkBundleAdminController
     const CONTROLLER_NAME = 'AdminMaintenance';
 
     /**
-     * @var FormInterface
-     * @Template("@PrestaShop/Admin/Configure/ShopParameters/maintenance.html.twig")
+     * @AdminSecurity("is_granted(['read', 'update', 'create', 'delete'], request.get('_legacy_controller'))")
```

Comment:
I figured out I approved this PR, but we have the same discussion as in #12098 ><

Reply:
So we agreed to go for "read" only. I need to update this PR.